packages/graphql_flutter/pubspec.yaml

```diff
@@ -15,6 +15,7 @@ dependencies:
   meta: ^1.7.0
   path_provider: ^2.0.1
   path: ^1.8.0
+  #connectivity_plus: ^5.0.0
```

Comment:
not sure what you want to do with this commit, but I think this should be folded in the previous one